src/components/PriceLine.tsx

```diff
@@ -120,7 +122,14 @@ const PriceLine = ({
                         />
                     )}
                 </div>
-                {title && <PriceLineTitle color={color} title={title} yAxiswidth={overlappedBarrierWidth} opacity={opacity} />}
+                {title && (
+                    <PriceLineTitle
+                        color={color}
+                        title={title}
+                        yAxiswidth={width + price_right_offset}
```

Comment:
This will not affect main barrier because we don't get any title for the main barrier from API.
And even if we received a title for it, **Theoretically**, and if the barrier would overlap the price, the title would be placed to the left of the barrier price like this:
<img width="1728" alt="Screenshot 2023-12-27 at 2 18 38 PM" src="https://github.com/binary-com/SmartCharts/assets/103177211/ce7c5256-f567-4d2e-a020-d5d98566d47a">